### ISSUE TITLE
Edit profile: Alert section missing information

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#52]
URL: https://hcdev.openplus.ca/en/user/203/edit
User Agent: Chrome, Firefox
Language: both

Issue Description: Alert section text does not seem to be done correctly and has information missing from visual users. Also, the visual information is not correct. There are a few issues with it. Please see the list of issues and corresponding expected results below:

1. There is a hidden heading h2 "Error message" which is only announced with screen readers.
2. Alert section has a Close button.
3. The error says "Password". It doesn't describe what the error actually is.

Steps to reproduce: Enter different passwords in Password and Confirm password fields

Expected results:

1. All content (headings) should be available to all users. Heading should not be hidden
2. Close button should be removed as alert sections don't need to be closed.
3. The error message should describe what the error is, like field is required or password fields do not match, etc.

Please follow WET format https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html

![image](https://user-images.githubusercontent.com/14280062/128871960-60d2e972-aaea-4d39-a9cf-8a1328a2cc63.png)

This is WET example:

![image](https://user-images.githubusercontent.com/14280062/128872007-7c5260dc-f007-4bd2-9b6e-f1cc0409f1b7.png)

**Comment by echouin** [comment]
From OP: ﻿the errors have been updated . For comparison, here is as screenshot .

![image](https://user-images.githubusercontent.com/88099454/157727515-b669b71f-d1b2-4551-9b76-c6ad3f4675c2.png)

**Comment by ndolter** [comment]
Re-tested in Chrome (v99) and NVDA and performed the following test from the Edit Profile page:
1. Entered a valid password in both the Username and Confirm Password fields, but made them to not match
2. Clicked Save
Issue: 
1. There is still a hidden heading h2 "Error message"
2. The same error is being reported.  The screen reader is not announcing 'The specified passwords do not match' (currently located under the 'Confirm Password' field.  It would be more informative for this message to be included in the error message in the alert, so that the screen reader users are provided with the same information as visual users.  

Please note the close button ('X') is no longer an issue on the Alert
![image](https://user-images.githubusercontent.com/68117281/161118837-89754033-6c55-4bef-907e-e9b2e86b1612.png)

**Comment by echouin** [comment]
this is now resoled. 
![image](https://github.com/user-attachments/assets/03e84fda-6a23-468a-b8e5-1b6af6c16e88)
